doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -150,6 +150,31 @@ Some examples for the output of SSL_CIPHER_description():
  ECDHE-RSA-AES256-GCM-SHA256 TLSv1.2 Kx=ECDH     Au=RSA  Enc=AESGCM(256) Mac=AEAD
  RSA-PSK-AES256-CBC-SHA384 TLSv1.0 Kx=RSAPSK   Au=RSA  Enc=AES(256)  Mac=SHA384
 
+=head1 RETURN VALUES
+
+SSL_CIPHER_get_name(), SSL_CIPHER_standard_name(), OPENSSL_cipher_name(),
+SSL_CIPHER_get_version() and SSL_CIPHER_description() return the corresponding
+value in null-terminated string for a specific cipher or "(NULL)"
+if the cipher is not found.
+
+SSL_CIPHER_get_bits() returns a positive integer or 0 if an error occurred.
+
+SSL_CIPHER_get_cipher_nid(), SSL_CIPHER_get_digest_nid(),
+SSL_CIPHER_get_kx_nid() and SSL_CIPHER_get_auth_nid() return NID values or
+B<NID_undef> if an error occurred.
```

Comment:
"NID value*s*" stings my eye a little, but that might be just me. I mean ping native English speakers if it should be "NID value"...

Reply:
NID value - it is a singular return.